index.js

```diff
@@ -15,8 +15,15 @@ function buildInspectorUrl(inspectorHost, inspectorPort, debugPort, fileToShow)
     hostname: inspectorHost || '127.0.0.1',
     port: inspectorPort,
     pathname: '/debug',
-    search: '?port=' + debugPort
+    query: {
+        port: debugPort
+    }
   };
+  // For backward compatibility, we'd like to keep arguments.
+  // But if arguments.length > 4, we take fifth argument as a debugHost.
+  if (buildInspectorUrl.arguments.length > 4 && buildInspectorUrl.arguments[4]) {
```

Comment:
You're right. `arguments` seems proper here instead of ` buildInspectorUrl.arguments`. 
And I thought `debugPort` and `debugHost` should be adjacent so that the function arguments look intuitive, therefore I didn't add it to the function signature.
But now I changed my mind because it doesn't look different much that `arguments[4]` and `debugHost` as the last argument. I'll add `debugHost` to the function signature.